sbin/prepareWorkspace.sh

```diff
@@ -477,19 +477,21 @@ checkingAndDownloadingFreeType() {
       return
     fi
 
-    local pngArg=""
-    if ./configure --help | grep "with-png"; then
-      pngArg="--with-png=no"
-    fi
-
     local freetypeEnv=""
     if [[ "${BUILD_CONFIG[OS_ARCHITECTURE]}" == "i686" ]] || [[ "${BUILD_CONFIG[OS_ARCHITECTURE]}" == "i386" ]]; then
       freetypeEnv="export CC=\"gcc -m32\""
     fi
 
+    eval "${freetypeEnv}" && bash ./autogen.sh 
```

Comment:
Should we check the return code of this? My decoupling it from the configure (which was run with a `&&` to ensure it wouldn't continue if the autogen failed) we've lost the exit code from this step which may lead to confusion in the logs.

Reply:
Done, now the script will exit if the autogen.sh script fails